### PR TITLE
[GLT-3538] updates to match MISO v1.31.0

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,48 @@ Home of the training documents for the
...
+bundle install
+```
+
+On MacOS, Bundler may fail to install libv8. To work around this, you can
+install `v8-315` using Homebrew (or otherwise) and then install the dependencies
+that use it using the following before trying `bundle install` again.
+
+```
+gem install libv8 -v '3.16.14.19' -- --with-system-v8
+gem install therubyracer -v '0.12.3' -- --with-v8-dir=/usr/local/opt/v8@3.15
```
Comment (callunity) [inline README.md:44]: Why does it work to install libv8@3.16 but declare v8-dir@3.15?

File: _includes/libraries-from-samples.md [hunk 2]
```diff
@@ -23,33 +23,40 @@ In this section, you will use the samples created already to create libraries.
...
 {% endif %}
-    * _Index Kit_: {{ site.index_kit }}
-    * _Index 1_ and _Index 2_ (if available): Select any combination of indices you wish.
-      Select different indices for each library. Selecting the same indices for two
-      different libraries would make it unwise to pool those two libraries
-      together later.
+    * _Index Kit_: `{{ site.index_kit }}`
+    * _Index 1_ and _Index 2_ (if available): Select any combination of indices
+      you wish. Select different indices for each library. Selecting the same
+      indices for two different libraries would make it unwise to pool those two
```
Comment (callunity) [inline _includes/libraries-from-samples.md:48]: Remind me - does MISO prevent the user from creating pools with colliding indices?

File: _includes/runs-trouble.md [hunk 3]
```diff
@@ -6,9 +6,9 @@
     * Add a service record as in <a href="#service-records">8: Adding Service Records to Sequencers</a>
 
 1. How do you assign single lane failures (if possible)?
```
Comment (callunity) [inline _includes/runs-trouble.md:8]: Could also add an item about assigning single run-library failures

**Review by callunity — APPROVED** [review]

**Review by avarsava — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (djcooke) [reply README.md:44]: This is referring to the system v8. I wasn't able to find a way to install 3.16 on the system, but 3.15 doesn't seem to have any compatibility issues with these gems.

File: _includes/libraries-from-samples.md (hunk 2 above)
Comment (djcooke) [reply _includes/libraries-from-samples.md:48]: It may - depends on configuration

File: _includes/runs-trouble.md (hunk 3 above)
Comment (djcooke) [reply _includes/runs-trouble.md]: Added

**Pull request merged**